src/runtime/RegExpObject.cpp

```diff
@@ -494,6 +506,113 @@ void RegExpObject::pushBackToRegExpMatchedArray(ExecutionState& state, ArrayObje
     }
 }
 
+bool RegExpObject::isRegExpObjectDontHaveAnyOwnPropertyWhichHasDefinedFromRegExpPrototype(ExecutionState& state, Object* obj)
+{
```

Comment:
Would you please simplify the name of this method?
It's somewhat too long to use..

Reply:
I changed function name